src/core/providers/accountProvider.ts

```diff
@@ -1,8 +1,8 @@
-import { IDappProvider } from 'types/dappProvider.types';
 import { emptyProvider } from './helpers/emptyProvider';
 import { CrossWindowProvider } from 'lib/sdkWebWalletCrossWindowProvider';
+import { IProvider } from 'core/providers/types/providerFactory.types';
 
-export type ProvidersType = IDappProvider | CrossWindowProvider;
+export type ProvidersType = IProvider | CrossWindowProvider;
```

Comment:
is the | needed ?

Reply:
yes...at least until the next PR when webWalletLogin will be refactored in the same way as extension provider login